docs/modules/ROOT/pages/list-of-camel-quarkus-extensions.adoc

```diff
@@ -312,49 +312,31 @@ Number of Camel data formats: 23 in 18 JAR artifacts (0 deprecated)
 == Camel Languages
 
 // languages: START
-Number of Camel languages: 11 in 5 JAR artifacts (0 deprecated)
+Number of Camel languages: 3 in 3 JAR artifacts (0 deprecated)
```

Comment:
Is it expected that extensions are removed from here? We should be listing all of the stuff that camel-quarkus supports.

Reply:
No, thanks for spotting it!